frontend/src/scenes/max/Max.tsx

```diff
@@ -51,11 +51,11 @@ export function Max({ tabId }: { tabId?: string }): JSX.Element {
     if (sidePanelOpen && selectedTab === SidePanelTab.Max && sidepanelConversationId === tabConversationId) {
```

Comment:
Out of scope but let's do it sooner than later, we should rename these files from `Max.etc` to something else